doc/en/example/parametrize.rst

```diff
@@ -411,8 +411,10 @@ is to be run with different sets of arguments for its three arguments:
 Running it results in some skips if we don't have all the python interpreters installed and otherwise runs all combinations (5 interpreters times 5 interpreters times 3 objects to serialize/deserialize)::
 
    . $ pytest -rs -q multipython.py
-   ...........................                                          [100%]
-   27 passed in 0.12 seconds
+   ...sss...sssssssss...sss...                                          [100%]
```

Comment:
this one needs a follow up

Reply:
@nicoddemus i dont see the need to change it for this release, but i'd like to review the interpreter matrix for the next release

Reply:
Fair enough, thanks. Created #3496 to track this.

Btw after this is out we should try to get the pre-commit and auto-formatter in `master` ASAP.

Reply:
im soooo on it ^^ - i'm currently like a dog trying to dig trough the wood of a castles gate